pegasus/sites.v3/advocacy.code.org/public/css/interactive_map.css

```diff
@@ -0,0 +1,29 @@
+#map {
+  width: 700px;
+  min-height: 425px;
+  float: left;
+  margin-right: -100px;
+}
+
+#infobox {
+  text-align: left;
+  float: right;
+}
+
+.info {
+  font-size: 18px;
+}
+
+#state-dropdown {
+  width: 70%;
+  height: 34px;
+  background-color: #e2e4e3;
+  color: #595959;
+  margin-bottom: 10px;
+}
+
+.info a:link, .info a:visited, .states-link a:link, .states-link a:visited {
+  color: #888;
+  cursor: pointer;
+  text-decoration: underline;
+}
```

Comment:
I wonder if it's better to layer over our changes in a fresh new `.css` file instead of copying & modifying.  If eventually we want to reference the base file + some modifications, it might be hard to reconstruct if we don't start that way.

Reply:
We're slightly unusual in this change because we do intend to pull the new changes to existing components back into those components before we go live, but for now I've indeed gone with a simple CSS override where easiest, along with some parameterisation of the base components too.